app/build.gradle

```diff
@@ -68,13 +68,13 @@ android {
 dependencies {
 
     // Import the Compose BOM
-    implementation platform('androidx.compose:compose-bom:2023.03.00')
+    implementation platform('androidx.compose:compose-bom:2023.04.01')
```

Comment:
05?

Reply:
not according to this page
https://developer.android.com/jetpack/compose/bom